web-client/src/views/Messages/MessageTable.tsx

```diff
@@ -142,19 +142,26 @@ export const MessageTable = connect<
             </div>
           </div>
         )}
-        <div className="overflow-x-auto" id={id}>
-          <div className="grid-row grid-gap">
-            {messageFilters.length > 0 && (
-              <div className="desktop:grid-col-8 tablet:grid-col-12 display-flex flex-align-center">
-                <TableFilters
-                  filters={messageFilters}
-                  onSelect={updateMessageFilterSequence}
-                ></TableFilters>
-              </div>
-            )}
-            {selectable && getCompleteAllButton()}
-          </div>
 
+        <div className="grid-row grid-gap">
+          {messageFilters.length > 0 && (
+            <div
+              className={
+                selectable
+                  ? 'desktop:grid-col-8 tablet:grid-col-12 display-flex flex-align-center' // To take into account the Complete All button
```

Comment:
Style needs to be dynamic to take into account whether the Complete All button appears or not (otherwise the layout of filters is messed up when the Complete All button doesn't appear)